Project-Aurora/Project-Aurora/Settings/Layers/AmbilightLayerHandler.cs

```diff
@@ -28,17 +28,17 @@ public enum AmbilightType
 
     public enum AmbilightCaptureType
     {
-        [Description("Everything")]
-        Everything = 0,
-
-        [Description("Main Monitor Only")]
-        MainMonitor = 1,
+        [Description("Entire Monitor")]
+        EntireMonitor = 0,
 
         [Description("Foreground Application")]
-        ForegroundApp = 2,
+        ForegroundApp,
 
         [Description("Specific Process")]
-        SpecificProcess = 3
+        SpecificProcess,
```

Comment:
I'd be wary about reordering this enum, as it means that people's existing settings will change about when they use the new version

Reply:
Should i set Coordinates to be '0' instead? The Everything option doesn't exist anymore.

Reply:
I decided to put the coordinates at 0. Would another way be better?

Reply:
Hmmm, I'm not sure. You could potentially set the default coords to -1 for all and have that result in it taking the entire screenshot so the behaviour is the same? How would you do all screens with the changes you've made?

Reply:
Setting the defaults to -1 won't work, as the coordinates are relative to your primary monitor. Negative ones are valid if you have monitors above or to the left of the main one. I tried to make this clearer by disabling the "Output ID" textbox whenever the user as any option other than "Entire Monitor" selected. Does that make sense?